internal/tfengine/tfengine_test.go

```diff
@@ -48,18 +48,40 @@ func TestExamples(t *testing.T) {
 				t.Fatalf("tfengine.Run(%q, %q) = %v", ex, tmp, err)
 			}
 
-			// Run plan on live dir to verify configs.
-			path := filepath.Join(tmp, "live")
+			// Run plan to verify configs.
+			fs, err := ioutil.ReadDir(tmp)
+			if err != nil {
+				t.Fatalf("ioutil.ReadDir = %v", err)
+			}
 
-			// Convert the configs not reference a GCS backend as the state bucket does not exist.
-			if err := ConvertToLocalBackend(path); err != nil {
-				t.Fatalf("ConvertToLocalBackend(%v): %v", path, err)
+			// Skip these dirs as they use data sources that are not available until dependencies have been deployed.
```

Comment:
Wouldn't this mean that we're no longer testing configs under these directories? What happens if we make a mistake and these generate invalid configs?

Reply:
We never tested them, only the 'live' dir. Since now we are flattening things live stuff will move to top level and get tested automatically.